src/SumoCoders/FrameworkCoreBundle/Resources/assets/js/Framework/Form.js

```diff
@@ -4,6 +4,7 @@ const locale = new Locale()
 
 export class Form {
   constructor (form) {
+    this.initForm()
```

Comment:
Can't the initForm code not come in the constructor?

Reply:
Wouldn't that make the constructor a bit messy?

Reply:
It's not like it's 10 lines of code, then I would start considering it, but in this case just 2.

Reply:
I agree with @jonasdekeukelaere but I wouldn't do it. If some code gets added later on that would mean refactoring the code _again_. Just calling these methods in the constructors is a lot better than it was before.

Reply:
i agree with @StijnVrolijk